### PR TITLE
refactor js code in schedule-panel js

### PR DESCRIPTION
**Pull request by fsi206914**
I'm working on the Azkaban Javascript Test initiative. By going through other standard front end projects and javascript lesson make ups, I know that 
1) .this part of code should be placed in **util/model**, for ease of test. 
2). schedule-panel.js is a **view** js, and it should only include DOM operations.
3). moving this cron-util code to date.js, since date.js is a util library file in Azkaban. schedulepanel.vm already imports date.js

Tested locally, and it should all work

### CODE REVIEW

**Comment by HappyRay** [comment]
Please clarify what "this" means in your commit description.

File: azkaban-web-server/src/web/js/azkaban/view/schedule-panel.js [hunk 1]
```diff
@@ -217,6 +195,9 @@ function updateOutput() {
 
 function updateExpression() {
   $('#nextRecurId').html("");
+
+  //Unix Cron use 0-6 as Sun--Sat, but Quartz use 1-7. Due to later.js only supporting Unix Cron, we have to make this transition.
+  //transformFromQuartzToUnixCron is defined in util/date.js
```
Comment (HappyRay) [inline azkaban-web-server/src/web/js/azkaban/view/schedule-panel.js]: The same comment is in the date.js already. What do you think about removing this to make the comment easier to maintain?

**Comment by HappyRay** [comment]
LGTM

File: azkaban-web-server/src/web/js/azkaban/view/schedule-panel.js (hunk 1 above)
Comment (fsi206914) [inline azkaban-web-server/src/web/js/azkaban/view/schedule-panel.js]: I intended to place it here, in order to let readers know why we need this transition. What do you think?

**Pull request merged**